python/pyspark/sql/connect/column.py

```diff
@@ -19,18 +19,59 @@
 
 
 import pyspark.sql.connect.proto as proto
+
```

Comment:
Should probably remove this since `pyspark` should be within the same import block.

Reply:
yes, will clean it up